src/types/entityTypes.ts

```diff
@@ -20,3 +21,16 @@ export interface EntityContext<T> extends InvocationContext {
      */
     df: DurableEntityContext<T>;
 }
+
+/**
+ * The return value of a call to `df.app.entity()`
+ */
+export interface RegisterEntityResult {
+    /**
+     * Returns a new `EntityId` instance with the specified key. Can be passed to entity operation APIs.
+     *
+     * @param key the id to uniquely identity an entity instance among all instances of the same class.
+     * @returns an `EntityId` instance using the registered entity name and the specified key.
+     */
+    getEntityId: (key: string) => EntityId;
+}
```

Comment:
Any reason why this has a uniquely different name?  The other types have a `-Input` suffix.

Reply:
Because this one isn't passed to one unique API, but can be used to create an `EntityId` that itself is used in multiple APIs (`signalEntity()`, `callEntity()`, `getEntityState()`, etc.). I figured it's easier to characterize this not by what's it's an input _to_, but rather what it's an output _of_. Admittedly, I didn't have particularly good names for these and I'm open for suggestions :)

Reply:
I see what you mean and realize that this is used differently. But it's also this type is "the same" as the other ones in that it is an output of the Function-registration APIs. So I'd recommend making their names uniform. I'd suggest just calling them `Activity`, `Orchestrator`, and `Entity` if possible.

Reply:
Done